index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Gowani Pump Supply</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Google Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
+
+    <!-- Style Sheet for Sticky Bootstrap Footer -->
+    <link href="css/sticky-footer.css" rel="stylesheet">
+
+    <!-- Animate CSS -->
+    <link rel="stylesheet" href="css/animate.css">
+
+    <!-- Custom Style Sheets -->
+    <link rel="stylesheet" href="css/style.css">
+
+  </head>
+
+  <body>
+
+  <nav class="navbar navbar-toggleable-sm navbar-inverse bg-faded fixed-top">
+
+  <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span>&#x2630;</span>
+  </button>
+
+  <a class="navbar-brand" href="#main" style="font-family: 'Open Sans', sans-serif;">Gowani Pump Supply</a>
+
+  <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <ul class="navbar-nav mr-auto">
+      <!-- Including this piece of code helps keep the next nav-item to the right -->
+    </ul>
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link btn btn-sm btn-info" href="#contact" role="button">
+          Contact
+        </a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link btn btn-sm btn-info coming_soon" role="button">
+          Request Service
+        </a>
+      </li>
+    </ul>
+  </div>
+  </nav>
+  
+  <section class="cover" id="main">
```

Comment:
The uses of `id=` is not really needed anywhere here, plus the community is trying to move towards using `class` over `id`.

Instead of setting an `id` on something like the `<h1>`, you can do:

```
.cover h1 {
  /* style here */
}
```